tests/integration/components/location-map-test.js

```diff
@@ -0,0 +1,24 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('location-map', 'Integration | Component | location map', {
+  integration: true
+});
+
+test('it renders', function(assert) {
```

Comment:
This is just the default generated, I plan to add a few tests around validating that the maps get rendered.